src/components/modals/Modal.vue

```diff
@@ -46,15 +60,7 @@ export default {
   box-shadow: 2px 2px 20px 1px #000;
   display: flex;
   flex-direction: column;
-  max-width: 60%;
-
-  .characters &,
-  .vote-history &,
-  .night-reference & {
-    max-height: 80%;
-    max-width: 80%;
-    overflow-y: auto;
-  }
+  overflow-y: auto;
```

Comment:
Please don't change the default layout for the modals, there's a good reason why most of them don't have `overflow-y: auto`: there are tooltips inside the modal that need to be visible beyond the edge of the modal.
![Bildschirmfoto 2021-02-10 um 21 21 16](https://user-images.githubusercontent.com/325521/107567424-efcffc00-6be5-11eb-88a7-b8734a0bbc4b.png)

Reply:
aha, will fix